docs/esp8266/general.rst

```diff
@@ -151,25 +151,26 @@ SSL/TLS limitations
 ~~~~~~~~~~~~~~~~~~~
 
 ESP8266 uses `axTLS <http://axtls.sourceforge.net/>`_ library, which is one
-of the smallest TLS libraries with the compatible licensing. However, it
+of the smallest TLS libraries with compatible licensing. However, it
 also has some known issues/limitations:
 
 1. No support for Diffie-Hellman (DH) key exchange and Elliptic-curve
-   cryptography (ECC). This means it can't work with sites which force
-   the use of these features (it works ok with classic RSA certifactes).
+   cryptography (ECC). This means it can't work with sites which require
+   the use of these features (it works ok with the typical sites that use
+   RSA certificates).
 2. Half-duplex communication nature. axTLS uses a single buffer for both
    sending and receiving, which leads to considerable memory saving and
    works well with protocols like HTTP. But there may be problems with
    protocols which don't follow classic request-response model.
 
-Besides axTLS own limitations, the configuration used for MicroPython is
+Besides axTLS's own limitations, the configuration used for MicroPython is
```

Comment:
Perhaps. I purposedly avoided that due to ambiguities in UK vs US spelling of possessive here.